system/wymeditor/plugins/embed/jquery.wymeditor.embed.js

```diff
@@ -58,6 +58,40 @@
             ]
         };
 
+        WYMeditor.XhtmlValidator._tags.video = {
+            "attributes":[
+                "width",
+                "height",
+                "autoplay",
+                "controls",
+                "loop",
+                "muted",
+                "poster",
+                "preload",
+                "src"
+            ]
+        };
+
+        WYMeditor.XhtmlValidator._tags.audio = {
+            "attributes":[
+                "autoplay",
+                "controls",
+                "loop",
+                "muted",
+                "preload",
+                "src"
+            ]
+        };
+
+        WYMeditor.XhtmlValidator._tags.source = {
+            "attributes":[
```

Comment:
there seems to be a srcset attribute as well
source: (:-p) https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source

Reply:
I saw that, but according to w3schools that was only valid when using `<picture>`, which we don't support anyway. But, reconsidering, might be better to just add this now and make it easier to add `<picture>` later.